pymagicc/core.py

```diff
@@ -777,7 +777,9 @@ def _format_config(self, config_dict):
 
     def _convert_out_config_flags_to_integers(self, config_dict):
         for key, value in config_dict.items():
-            if key.startswith("out") and key != "out_ascii_binary":
+            # Ignore all out_x_vars variables as that should be a list of strings
+            # Note that there is a variable OUT_ALLOWANYDYNAMICVARS which should be in integer
```

Comment:
What's `OUT_ALLOWANYDYNAMICVARS`?

Reply:
Malte has added a variable to force the calculation of some of the summary variables. Previously these were only calculated when OUT_KEYDATA_X was set. This isn't in master yet

Reply:
ok so a discussion for the MAGICC repo then